Sources/PointFree/Account/Change.swift

```diff
@@ -0,0 +1,392 @@
+          .map { $0.right?.count ?? 0 }
+        ])
+
+      return invitesAndTeammates
+        .sequential
+        .flatMap { middleware(conn.map(const(user .*. $0.reduce(1, +) .*. conn.data.second))) }
+    }
+}
+
+let subscriptionChangeShowView = View<(Stripe.Subscription, Database.User, Int)> { subscription, currentUser, seatsTaken -> Node in
+
+  gridRow([
+    gridColumn(sizes: [.mobile: 12, .desktop: 8], [style(margin(leftRight: .auto))], [
+      div(
+        [`class`([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])])],
+        titleRowView.view(subscription)
+          <> formRowView.view((subscription, seatsTaken))
+          <> cancelRowView.view(subscription)
+      )
+      ])
+    ])
+}
+
+private let titleRowView = View<Stripe.Subscription> { subscription in
+  gridRow([`class`([Class.padding([.mobile: [.bottom: 2]])])], [
+    gridColumn(sizes: [.mobile: 12], [
+      div([
+        h1([`class`([Class.pf.type.responsiveTitle2])], ["Modify subscription"]),
+        p([
+          "You are currently enrolled in the ", strong([text(subscription.plan.name)]), " plan. ",
+          "Your subscription will ",
+          subscription.isRenewing ? "renew" : "end",
+          " on ",
+          strong([text(dateFormatter.string(from: subscription.currentPeriodEnd))]),
+          ".",
+          subscription.isRenewing
+            ? ""
+            : " Reactivate your subscription by submitting the form below."
+          ]),
+        ])
+      ])
+    ])
+}
+
+private let formRowView = View<(Stripe.Subscription, Int)> { subscription, seatsTaken -> Node in
+
+  return form(
+    [action(path(to: .account(.subscription(.change(.update(nil)))))), method(.post), `class`([Class.margin([.mobile: [.bottom: 3]])])],
+    changeSeatsRowView.view((subscription, seatsTaken))
+      <> changeBillingIntervalRowView.view(subscription)
+      <> [
+        hr([`class`([Class.pf.components.divider])])
+    ]
+  )
+}
+
+private let changeSeatsRowView = View<(Stripe.Subscription, Int)> { subscription, seatsTaken -> Node in
+
+  let pricing = PointFree.pricing(for: subscription)
+  let subtitle = pricing.isIndividual
+    ? "Change to a team subscription?"
+    : "Add or remove seats?"
+  let description: [Node] = pricing.isIndividual
+    ? ["Specify the total number of seats you’d like."]
+    : [
+      "You are currently using ", strong([text(String(seatsTaken)), " of ",
+      strong([text(String(subscription.quantity))]), " seats"]), " available."
+  ]
+
+  return gridRow([`class`([Class.padding([.mobile: [.bottom: 2]])])], [
+    gridColumn(sizes: [.mobile: 12], [
+      h3([`class`([Class.pf.type.responsiveTitle4])], [text(subtitle)]),
+      p(description + [" ",
+        """
+        Additional costs will be billed immediately, prorated against the remaining time of
+        the current billing period.
+        """
+        ]),
+      input([
+        type(.number),
+        min(max(1, seatsTaken)),
```

Comment:
if they choose `1` does it change them to individual? or should we use `2` as the minimum?

Reply:
oh I see, the individual vs team is only a distinction we make based on quantity. cool!